<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-8BD0QAM" ignored="0" name="Test" tests="5" failures="1" timestamp="2022-12-16T01:03:59 IST" time="7.985" errors="0">
  <testcase name="deleteuser" time="6.511" classname="testcases.TC005_Delete_Record">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at testcases.TC005_Delete_Record.deleteuser(TC005_Delete_Record.java:37)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- deleteuser -->
  <testcase name="updateuser" time="1.179" classname="testcases.TC004_Update_User"/>
  <testcase name="getsingleuser" time="0.053" classname="testcases.TC002_Get_Single_user"/>
  <testcase name="getAllUsers" time="0.068" classname="testcases.TC001_Get_All_Users"/>
  <testcase name="adduser" time="0.089" classname="testcases.TC003_Add_User"/>
</testsuite> <!-- Test -->
